src/WebSockets.jl

```diff
@@ -71,17 +71,17 @@ function send_fragment(ws::WebSocket, islast::Bool, data::Array{Uint8}, opcode=0
   b1::Uint8 = (islast ? 0b1000_0000 : 0b0000_0000) | opcode
   if l <= 125
     write(ws.socket,b1)
-    write(ws.socket,uint8(l))
+    write(ws.socket,convert(Uint8,l))
```

Comment:
`Uint` types are here for now, until question about type deprecation is resolved, [julia-#11200](https://github.com/JuliaLang/julia/issues/11200).